jobs/jobs/create_job_funcs.py

```diff
@@ -174,12 +187,27 @@ async def create_extraction_annotation_job(
     )
 
     files_data = utils.delete_duplicates(files_data)
-    if not files_data:
+    if not (
```

Comment:
same, just write using `if not files_data or not extraction_job_input.previous_jobs` expression

Reply:
what we have to check:
1. files/datasets or previous_jobs is filled
2. in one request we can fill only files/datasets or previous_jobs, but not both

Reply:
so if I'm right let's leave it as it is